[2016-09-19 10:19:05,947] ERROR [play-thread-1] play.Logger.niceThrowable(Logger.java:604) - 

@71e8mka1a
Internal Server Error (500) for request GET /buyer/account3

No route found (In /app/views/UserCenter/buyerAccount3.html around line 50)
No route able to display file /public/images/warn.png (file not found) was found.

play.exceptions.NoRouteFoundException: No route found
	at play.templates.BaseTemplate.throwException(BaseTemplate.java:80)
	at play.templates.GroovyTemplate.internalRender(GroovyTemplate.java:270)
	at play.templates.Template.render(Template.java:26)
	at play.templates.GroovyTemplate.render(GroovyTemplate.java:208)
	at play.mvc.results.RenderTemplate.<init>(RenderTemplate.java:24)
	at play.mvc.Controller.renderTemplate(Controller.java:676)
	at play.mvc.Controller.renderTemplate(Controller.java:656)
	at play.mvc.Controller.render(Controller.java:713)
	at controllers.UserCenter.buyerAccount3(UserCenter.java:1064)
	at play.mvc.ActionInvoker.invokeWithContinuation(ActionInvoker.java:524)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:475)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:451)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:446)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:160)
	at Invocation.HTTP Request(Play!)
[2016-09-19 10:25:40,546] ERROR [jobs-thread-5] play.Logger.niceThrowable(Logger.java:604) - 

@71e8mka1b
Error during job execution (jobs.TaskKeeper)

Compilation error (In /app/controllers/UserCenter.java around line 1160)
The file /app/controllers/UserCenter.java could not be compiled. Error raised is : The method checkBeforeSaveBuyerAccount(BuyerAccount, User) in the type UserCenter is not applicable for the arguments (BuyerAccount3, User)

play.exceptions.CompilationException: The method checkBeforeSaveBuyerAccount(BuyerAccount, User) in the type UserCenter is not applicable for the arguments (BuyerAccount3, User)
	at play.classloading.ApplicationCompiler$2.acceptResult(ApplicationCompiler.java:256)
	at org.eclipse.jdt.internal.compiler.Compiler.handleInternalException(Compiler.java:678)
	at org.eclipse.jdt.internal.compiler.Compiler.compile(Compiler.java:522)
	at play.classloading.ApplicationCompiler.compile(ApplicationCompiler.java:292)
	at play.classloading.ApplicationClasses$ApplicationClass.compile(ApplicationClasses.java:289)
	at play.classloading.ApplicationClassloader.detectChanges(ApplicationClassloader.java:326)
	at play.Play.detectChanges(Play.java:636)
	at play.Invoker$Invocation.init(Invoker.java:199)
	at play.jobs.Job.call(Job.java:185)
	at Invocation.Job(Play!)
[2016-09-19 10:25:40,737] ERROR [jobs-thread-3] play.Logger.niceThrowable(Logger.java:604) - 

@71e8mka1d
Error during job execution (jobs.TaskKeeper2)

Compilation error (In /app/controllers/UserCenter.java around line 1160)
The file /app/controllers/UserCenter.java could not be compiled. Error raised is : The method checkBeforeSaveBuyerAccount(BuyerAccount, User) in the type UserCenter is not applicable for the arguments (BuyerAccount3, User)

play.exceptions.CompilationException: The method checkBeforeSaveBuyerAccount(BuyerAccount, User) in the type UserCenter is not applicable for the arguments (BuyerAccount3, User)
	at play.classloading.ApplicationCompiler$2.acceptResult(ApplicationCompiler.java:256)
	at org.eclipse.jdt.internal.compiler.Compiler.handleInternalException(Compiler.java:678)
	at org.eclipse.jdt.internal.compiler.Compiler.compile(Compiler.java:522)
	at play.classloading.ApplicationCompiler.compile(ApplicationCompiler.java:292)
	at play.classloading.ApplicationClasses$ApplicationClass.compile(ApplicationClasses.java:289)
	at play.classloading.ApplicationClassloader.detectChanges(ApplicationClassloader.java:326)
	at play.Play.detectChanges(Play.java:636)
	at play.Invoker$Invocation.init(Invoker.java:199)
	at play.jobs.Job.call(Job.java:185)
	at Invocation.Job(Play!)
[2016-09-19 16:23:13,419] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:232) - =======>Server Fatal Error<======
[2016-09-19 16:23:13,419] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:233) - 
### Error updating database.  Cause: com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'gender' in 'field list'
### The error may involve models.mappers.BuyerAccountMapper3.insert-Inline
### The error occurred while setting parameters
### SQL: insert into buyer_account(nick,gender,platform,consignee,state,city,region,address,mobile,user_id,status,create_time,modify_time) values(?,?,?,?,?,?,?,?,?,?,?,?,?)
### Cause: com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'gender' in 'field list'
org.apache.ibatis.exceptions.PersistenceException: 
### Error updating database.  Cause: com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'gender' in 'field list'
### The error may involve models.mappers.BuyerAccountMapper3.insert-Inline
### The error occurred while setting parameters
### SQL: insert into buyer_account(nick,gender,platform,consignee,state,city,region,address,mobile,user_id,status,create_time,modify_time) values(?,?,?,?,?,?,?,?,?,?,?,?,?)
### Cause: com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'gender' in 'field list'
	at org.apache.ibatis.exceptions.ExceptionFactory.wrapException(ExceptionFactory.java:26)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.update(DefaultSqlSession.java:154)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.insert(DefaultSqlSession.java:141)
	at org.apache.ibatis.binding.MapperMethod.execute(MapperMethod.java:51)
	at org.apache.ibatis.binding.MapperProxy.invoke(MapperProxy.java:52)
	at com.sun.proxy.$Proxy90.insert(Unknown Source)
	at models.BuyerAccount3.save(BuyerAccount3.java:207)
	at controllers.UserCenter.saveBuyerAccount3(UserCenter.java:1165)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at play.mvc.ActionInvoker.invokeWithContinuation(ActionInvoker.java:524)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:475)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:451)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:446)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:160)
	at play.server.PlayHandler$NettyInvocation.execute(PlayHandler.java:267)
	at play.Invoker$Invocation$1.apply(Invoker.java:290)
	at play.Invoker$Invocation$1.apply(Invoker.java:287)
	at play.db.jpa.JPA.withTransaction(JPA.java:258)
	at play.db.jpa.JPA.withinFilter(JPA.java:217)
	at play.db.jpa.JPAPlugin$TransactionalFilter.withinFilter(JPAPlugin.java:299)
	at play.Invoker$Invocation.withinFilter(Invoker.java:271)
	at play.Invoker$Invocation.run(Invoker.java:287)
	at play.server.PlayHandler$NettyInvocation.run(PlayHandler.java:245)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:98)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:206)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918)
	at java.lang.Thread.run(Thread.java:662)
Caused by: com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'gender' in 'field list'
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:39)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:27)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:513)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:389)
	at com.mysql.jdbc.Util.getInstance(Util.java:372)
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:980)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3835)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3771)
	at com.mysql.jdbc.MysqlIO.sendCommand(MysqlIO.java:2435)
	at com.mysql.jdbc.MysqlIO.sqlQueryDirect(MysqlIO.java:2582)
	at com.mysql.jdbc.ConnectionImpl.execSQL(ConnectionImpl.java:2535)
	at com.mysql.jdbc.PreparedStatement.executeInternal(PreparedStatement.java:1911)
	at com.mysql.jdbc.PreparedStatement.execute(PreparedStatement.java:1203)
	at com.mchange.v2.c3p0.impl.NewProxyPreparedStatement.execute(NewProxyPreparedStatement.java:1392)
	at sun.reflect.GeneratedMethodAccessor28.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at org.apache.ibatis.logging.jdbc.PreparedStatementLogger.invoke(PreparedStatementLogger.java:62)
	at com.sun.proxy.$Proxy20.execute(Unknown Source)
	at org.apache.ibatis.executor.statement.PreparedStatementHandler.update(PreparedStatementHandler.java:44)
	at org.apache.ibatis.executor.statement.RoutingStatementHandler.update(RoutingStatementHandler.java:69)
	at org.apache.ibatis.executor.ReuseExecutor.doUpdate(ReuseExecutor.java:50)
	at org.apache.ibatis.executor.BaseExecutor.update(BaseExecutor.java:105)
	at org.apache.ibatis.executor.CachingExecutor.update(CachingExecutor.java:71)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.update(DefaultSqlSession.java:152)
	... 32 more
[2016-09-19 16:23:37,019] ERROR [play-thread-1] play.Logger.niceThrowable(Logger.java:604) - 

@71e8mka1f
Internal Server Error (500) for request GET /buyer/account3

Oops: OutOfMemoryError
An unexpected error occured caused by exception OutOfMemoryError: PermGen space

play.exceptions.UnexpectedException: Unexpected Error
	at play.Invoker$Invocation.onException(Invoker.java:245)
	at play.Invoker$Invocation.run(Invoker.java:305)
	at Invocation.HTTP Request(Play!)
Caused by: java.lang.OutOfMemoryError: PermGen space
	at java.lang.ClassLoader.defineClass1(Native Method)
	at java.lang.ClassLoader.defineClassCond(ClassLoader.java:631)
	at java.lang.ClassLoader.defineClass(ClassLoader.java:615)
	at play.classloading.ApplicationClassloader.loadApplicationClass(ApplicationClassloader.java:152)
	at play.classloading.ApplicationClassloader.getAllClasses(ApplicationClassloader.java:433)
	at play.Play.start(Play.java:527)
	at play.Play.detectChanges(Play.java:653)
	at play.Invoker$Invocation.init(Invoker.java:199)
	... 1 more
[2016-09-19 16:23:37,019] ERROR [play-thread-1] play.Logger.niceThrowable(Logger.java:604) - Error during the 500 response generation
java.lang.RuntimeException: java.lang.OutOfMemoryError: PermGen space
	at play.templates.BaseTemplate.throwException(BaseTemplate.java:93)
	at play.templates.GroovyTemplate.internalRender(GroovyTemplate.java:285)
	at play.templates.Template.render(Template.java:26)
	at play.templates.GroovyTemplate.render(GroovyTemplate.java:208)
	at play.server.PlayHandler.serve500(PlayHandler.java:792)
	at Invocation.HTTP Request(Play!)
Caused by: java.lang.OutOfMemoryError: PermGen space
	at sun.misc.Unsafe.defineClass(Native Method)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.io.ObjectStreamClass.getSerializableConstructor(ObjectStreamClass.java:1322)
	at java.io.ObjectStreamClass.access$1500(ObjectStreamClass.java:50)
	at java.io.ObjectStreamClass$2.run(ObjectStreamClass.java:435)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.io.ObjectStreamClass.<init>(ObjectStreamClass.java:411)
	at java.io.ObjectStreamClass.lookup(ObjectStreamClass.java:308)
	at java.io.ObjectStreamClass.initNonProxy(ObjectStreamClass.java:545)
	at java.io.ObjectInputStream.readNonProxyDesc(ObjectInputStream.java:1580)
	at java.io.ObjectInputStream.readClassDesc(ObjectInputStream.java:1493)
	at java.io.ObjectInputStream.readNonProxyDesc(ObjectInputStream.java:1580)
	at java.io.ObjectInputStream.readClassDesc(ObjectInputStream.java:1493)
	at java.io.ObjectInputStream.readOrdinaryObject(ObjectInputStream.java:1729)
	at java.io.ObjectInputStream.readObject0(ObjectInputStream.java:1326)
	at java.io.ObjectInputStream.readObject(ObjectInputStream.java:348)
	at java.util.HashMap.readObject(HashMap.java:1029)
	at java.io.ObjectStreamClass.invokeReadObject(ObjectStreamClass.java:969)
	at java.io.ObjectInputStream.readSerialData(ObjectInputStream.java:1852)
	at java.io.ObjectInputStream.readOrdinaryObject(ObjectInputStream.java:1756)
	at java.io.ObjectInputStream.readObject0(ObjectInputStream.java:1326)
	at java.io.ObjectInputStream.readObject(ObjectInputStream.java:348)
	at play.utils.Java.deserialize(Java.java:361)
[2016-09-19 16:33:11,159] ERROR [play-thread-1] com.aton.base.BaseController.handleWrongInput(BaseController.java:291) - Input wrong: Incorrect value
[2016-09-19 16:35:18,542] ERROR [jobs-thread-3] play.Logger.niceThrowable(Logger.java:604) - 

@71eak4hf5
Error during job execution (jobs.TaskKeeper2)

Compilation error (In /app/controllers/UserCenter.java around line 1498)
The file /app/controllers/UserCenter.java could not be compiled. Error raised is : Duplicate method accountTaskStatus(BuyerTaskSearchVo) in type UserCenter

play.exceptions.CompilationException: Duplicate method accountTaskStatus(BuyerTaskSearchVo) in type UserCenter
	at play.classloading.ApplicationCompiler$2.acceptResult(ApplicationCompiler.java:256)
	at org.eclipse.jdt.internal.compiler.Compiler.compile(Compiler.java:480)
	at play.classloading.ApplicationCompiler.compile(ApplicationCompiler.java:292)
	at play.classloading.ApplicationClasses$ApplicationClass.compile(ApplicationClasses.java:289)
	at play.classloading.ApplicationClassloader.detectChanges(ApplicationClassloader.java:326)
	at play.Play.detectChanges(Play.java:636)
	at play.Invoker$Invocation.init(Invoker.java:199)
	at play.jobs.Job.call(Job.java:185)
	at Invocation.Job(Play!)
[2016-09-19 16:35:18,735] ERROR [jobs-thread-4] play.Logger.niceThrowable(Logger.java:604) - 

@71eak4hf7
Error during job execution (jobs.TaskKeeper)

Compilation error (In /app/controllers/UserCenter.java around line 1498)
The file /app/controllers/UserCenter.java could not be compiled. Error raised is : Duplicate method accountTaskStatus(BuyerTaskSearchVo) in type UserCenter

play.exceptions.CompilationException: Duplicate method accountTaskStatus(BuyerTaskSearchVo) in type UserCenter
	at play.classloading.ApplicationCompiler$2.acceptResult(ApplicationCompiler.java:256)
	at org.eclipse.jdt.internal.compiler.Compiler.compile(Compiler.java:480)
	at play.classloading.ApplicationCompiler.compile(ApplicationCompiler.java:292)
	at play.classloading.ApplicationClasses$ApplicationClass.compile(ApplicationClasses.java:289)
	at play.classloading.ApplicationClassloader.detectChanges(ApplicationClassloader.java:326)
	at play.Play.detectChanges(Play.java:636)
	at play.Invoker$Invocation.init(Invoker.java:199)
	at play.jobs.Job.call(Job.java:185)
	at Invocation.Job(Play!)
[2016-09-19 16:45:15,912] ERROR [play-thread-1] com.aton.base.secure.Secure.checkAuthentication(Secure.java:92) - Login admin not exists,admin_id=null
[2016-09-19 17:16:21,925] ERROR [play-thread-1] null - 

@71eak4hf9
Internal Server Error (500) for request GET /admin3

Oops: OutOfMemoryError
An unexpected error occured caused by exception OutOfMemoryError: PermGen space

play.exceptions.UnexpectedException: Unexpected Error
	at play.Invoker$Invocation.onException(Invoker.java:245)
	at play.Invoker$Invocation.run(Invoker.java:305)
	at Invocation.HTTP Request(Play!)
Caused by: java.lang.OutOfMemoryError: PermGen space
	at java.lang.ClassLoader.defineClass1(Native Method)
	at java.lang.ClassLoader.defineClassCond(ClassLoader.java:631)
	at java.lang.ClassLoader.defineClass(ClassLoader.java:615)
	at play.classloading.ApplicationClassloader.loadApplicationClass(ApplicationClassloader.java:152)
	at play.classloading.ApplicationClassloader.getAllClasses(ApplicationClassloader.java:433)
	at play.Play.start(Play.java:527)
	at play.Play.detectChanges(Play.java:653)
	at play.Invoker$Invocation.init(Invoker.java:199)
	... 1 more
[2016-09-19 17:16:23,731] ERROR [play-thread-1]  - Error during the 500 response generation
java.lang.OutOfMemoryError: PermGen space
	at java.lang.ClassLoader.defineClass1(Native Method)
	at java.lang.ClassLoader.defineClassCond(ClassLoader.java:631)
	at java.lang.ClassLoader.defineClass(ClassLoader.java:615)
	at play.classloading.ApplicationClassloader.loadApplicationClass(ApplicationClassloader.java:152)
	at play.classloading.ApplicationClassloader.loadClass(ApplicationClassloader.java:82)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:247)
	at play.classloading.ApplicationClasses.getAssignableClasses(ApplicationClasses.java:71)
	at play.classloading.ApplicationClassloader.getAssignableClasses(ApplicationClassloader.java:459)
	at play.templates.GroovyTemplateCompiler.endTag(GroovyTemplateCompiler.java:360)
	at play.templates.TemplateCompiler.generate(TemplateCompiler.java:93)
	at play.templates.TemplateCompiler.compile(TemplateCompiler.java:15)
	at play.templates.GroovyTemplateCompiler.compile(GroovyTemplateCompiler.java:42)
	at play.templates.TemplateCompiler.compile(TemplateCompiler.java:28)
	at play.templates.TemplateLoader.load(TemplateLoader.java:84)
	at play.templates.TemplateLoader.load(TemplateLoader.java:179)
	at play.server.PlayHandler.serve500(PlayHandler.java:792)
	at play.server.PlayHandler$NettyInvocation.run(PlayHandler.java:247)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:98)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:206)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918)
	at java.lang.Thread.run(Thread.java:662)
[2016-09-19 17:31:47,389] ERROR [jobs-thread-4] play.Logger.niceThrowable(Logger.java:604) - 

@71eb1fchb
Error during job execution (jobs.TaskKeeper)

Compilation error (In /app/models/BuyerTask3.java around line 306)
The file /app/models/BuyerTask3.java could not be compiled. Error raised is : BuyerAccountMapper cannot be resolved to a type

play.exceptions.CompilationException: BuyerAccountMapper cannot be resolved to a type
	at play.classloading.ApplicationCompiler$2.acceptResult(ApplicationCompiler.java:256)
	at org.eclipse.jdt.internal.compiler.Compiler.handleInternalException(Compiler.java:678)
	at org.eclipse.jdt.internal.compiler.Compiler.compile(Compiler.java:522)
	at play.classloading.ApplicationCompiler.compile(ApplicationCompiler.java:292)
	at play.classloading.ApplicationClasses$ApplicationClass.compile(ApplicationClasses.java:289)
	at play.classloading.ApplicationClassloader.detectChanges(ApplicationClassloader.java:326)
	at play.Play.detectChanges(Play.java:636)
	at play.Invoker$Invocation.init(Invoker.java:199)
	at play.jobs.Job.call(Job.java:185)
	at Invocation.Job(Play!)
[2016-09-19 17:31:47,503] ERROR [jobs-thread-8] play.Logger.niceThrowable(Logger.java:604) - 

@71eb1fchd
Error during job execution (jobs.TaskKeeper2)

Compilation error (In /app/models/BuyerTask3.java around line 306)
The file /app/models/BuyerTask3.java could not be compiled. Error raised is : BuyerAccountMapper cannot be resolved to a type

play.exceptions.CompilationException: BuyerAccountMapper cannot be resolved to a type
	at play.classloading.ApplicationCompiler$2.acceptResult(ApplicationCompiler.java:256)
	at org.eclipse.jdt.internal.compiler.Compiler.handleInternalException(Compiler.java:678)
	at org.eclipse.jdt.internal.compiler.Compiler.compile(Compiler.java:522)
	at play.classloading.ApplicationCompiler.compile(ApplicationCompiler.java:292)
	at play.classloading.ApplicationClasses$ApplicationClass.compile(ApplicationClasses.java:289)
	at play.classloading.ApplicationClassloader.detectChanges(ApplicationClassloader.java:326)
	at play.Play.detectChanges(Play.java:636)
	at play.Invoker$Invocation.init(Invoker.java:199)
	at play.jobs.Job.call(Job.java:185)
	at Invocation.Job(Play!)
[2016-09-19 17:32:47,786] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:232) - =======>Server Fatal Error<======
[2016-09-19 17:32:47,786] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:233) - 
java.lang.NullPointerException
	at models.BuyerTask3.isSexTaking(BuyerTask3.java:314)
	at controllers.TaskCenter.isTakeTasks3(TaskCenter.java:557)
	at controllers.TaskCenter.listTasks3(TaskCenter.java:488)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at play.mvc.ActionInvoker.invokeWithContinuation(ActionInvoker.java:524)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:475)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:451)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:446)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:160)
	at play.server.PlayHandler$NettyInvocation.execute(PlayHandler.java:267)
	at play.Invoker$Invocation$1.apply(Invoker.java:290)
	at play.Invoker$Invocation$1.apply(Invoker.java:287)
	at play.db.jpa.JPA.withTransaction(JPA.java:258)
	at play.db.jpa.JPA.withinFilter(JPA.java:217)
	at play.db.jpa.JPAPlugin$TransactionalFilter.withinFilter(JPAPlugin.java:299)
	at play.Invoker$Invocation.withinFilter(Invoker.java:271)
	at play.Invoker$Invocation.run(Invoker.java:287)
	at play.server.PlayHandler$NettyInvocation.run(PlayHandler.java:245)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:98)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:206)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918)
	at java.lang.Thread.run(Thread.java:662)
[2016-09-20 08:52:24,839] ERROR [main] play.Logger.error(Logger.java:426) - Could not bind on port 80
